setup.py

```diff
@@ -153,7 +150,13 @@ def get_extras_require() -> Dict[str, List[str]]:
             "botorch>=0.4.0 ; python_version>'3.6'",
             "fastai",
         ],
-        "tests": ["fakeredis", "pytest"],
+        "tests": [
+            # TODO(c-bata): Remove Cython after removing version constraints of sklearn
+            # from 'optional' in extra_requires.
+            "Cython; python_version > '3.8'",
```

Comment:
According to the comment, this line should be moved to `optional`. The installation of `optional` actually failed due to the following error:

```
ModuleNotFoundError: Please install Cython with a version >= 0.28.5 in order to build a scikit-learn from source.
```

Reply:
Okay. I'm actually not sure we should force all Python 3.9 users to install Cython, but it's acceptable to add it in `optional` dependencies.